docs/en/observability/apm/security/apm-agents/api-keys.asciidoc

```diff
@@ -78,9 +78,11 @@ The Applications UI has a built-in workflow that you can use to easily create an
 Only API keys created in the Applications UI will show up here.
 
 Using a superuser account, or a user with the role created in the previous step,
-open {kib} and navigate to **{observability}** → **Applications** → **Settings** → **Agent keys**.
+open {kib}. Find **Applications** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. Go to **Settings** → **Agent keys**.
 Enter a name for your API key and select at least one privilege.
 
+//QUESTION FOR COLLEEN OR BRANDON: In the classic menu, I still see APM not "Applications". You can't find "applications" through the global search in classic. You need to search for apm. What a mess. Should I tell users to search for "apm"? (This applies to all places where I use this convention.)
```

Comment:
I think this is a bug. Surely it's not intentional? It looks like this in the main menu (when using the "Classic" view:

![image](https://github.com/user-attachments/assets/07f6cd7f-175e-4195-b22e-954ec99e4964)

But when you go into Observability, it looks like this:

![image](https://github.com/user-attachments/assets/63806159-c9bb-4fdc-a782-7f2384d5e931)

Reply:
There's an [open issue](https://github.com/elastic/observability-dev/issues/3719#issuecomment-2412044925) for the difference between the the Platform nav (first screenshot) and Observability nav (second screenshot). It doesn't mention search, but we can mention it in that issue, too.

Reply:
@colleenmcginnis I've added a comment to the issue so they remember to fix the search (just in case it's not fixed automagically when they fix the other problem).

What do you think I should do for the docs now? Just leave it as"Applications" and hope users who are using classic will know how to navigate?